### ISSUE TITLE
Thoughts on why this tool is useful

### ISSUE THREAD

**Issue by houlong666** [mshumer/gpt-prompt-engineer#6]
After reading the code, I feel that the idea of ​​this tool is to let gpt generate multiple candidate prompts, and then compare the results of multiple prompts. The results are still judged by gpt. Therefore, gpt is both an athlete and a referee, so why is it effective? Personal understanding may come from two aspects:
1. De-randomness: In this way, the prompt with the highest probability is selected to some extent. The essence is like de-randomization
2. Through different contexts, let gpt execute logic differently when it is an athlete and a referee, so it is effective

I want to ask if my understanding is correct, and if there are other reasons in it

**Comment by Devesh-N** [comment]
@houlong666 Yeah that's what I was thinking too! There is no real use for this project. #Overrated!

**Comment by Devesh-N** [comment]
> How about you just use it first and then comment on its usefulness based on the outcome?

@Phq-art I did, and my review is the same

**Comment by Phq-art** [comment]
Not that I care too much about your review since you just can not use it if you don't find it useful (?)
I get where you guys are coming from - a system that serves both as player and referee  being flawed - still I am having great results.
Besides all that I feel it's a big disrespect to the devs (which I am not one of) of this little project to come up and comment "no real use for this project #overrated". Where you are so wrong obviously. If you can't see it and prematurily judge that's a big L
